client/src/nl/sensorlab/videowall/animation/baseanimations/SortingAnimation.java

```diff
@@ -0,0 +1,105 @@
+package nl.sensorlab.videowall.animation.baseanimations;
+import java.util.ArrayList;
+
+import com.cleverfranke.util.PColor;
+
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.BozoSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.BubbleSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.CocktailSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.GnomeSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.OddEvenSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.OptimizedBubbleSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.Sort;
+import processing.core.PApplet;
+import processing.core.PGraphics;
+
+/**
+ * Visualization of different sorting methods
+ */
+public class SortingAnimation extends BaseAnimation {
+
+	private ArrayList<Sort> sortingmethods;
+	private int[] sortingColors = {}; // <-- Different colors for each sorting methods (just to make it more clear)
+
+
+	// Settings
+	private int sortingIndex = 0; // <-- Which methods to select
+	private int amountColumns = 12;
+	
+	// Update step
```

Comment:
Can you change the names for the method counters to something more understandable? I think there's time keeping between each sorting steps, and between methods. Perhaps something like sortStepElapsedTime or something?

Make clear also what the time base is. It is seconds, milliseconds, frames, eons?

Also make constants for the time final static ints, e.g. ```private static final int SORT_STEP_DURATION```